### ISSUE TITLE
Sqlite3 doen't install into dory-node android app

### ISSUE THREAD

**Issue by scidran** [TryGhost/node-sqlite3#949]
Hy.
I'm testing a proget with dory node (app into google play from tempage.io).
I've installed into a Samsung Tab A10.1 with android 7.0 and kernel 3.18.14, the gekko project from git hub.
Then i used:
`npm install --only=production`
But there was error to install sqlite3.
I tryed to install itmanually with
`npm install sqlite3`
But i have error
I attach this log
[npm-debug (npm bugs sqlite3).log](https://github.com/mapbox/node-sqlite3/files/1792911/npm-debug.npm.bugs.sqlite3.log)
[npm-debug (npm install --build-from-source).log](https://github.com/mapbox/node-sqlite3/files/1792912/npm-debug.npm.install.--build-from-source.log)
[npm-debug (npm install sqlite3 --loglevel=info).log](https://github.com/mapbox/node-sqlite3/files/1792913/npm-debug.npm.install.sqlite3.--loglevel.info.log)
Before try to root my Tablet, i would like to know if there is an other solution...
Thanks

**Comment by scidran** [comment]
I'm trying to find different solution for the problem... I tried also to root my tablet but I had issue also with this procedure because my tablet is unsupported yet.
I read carefully the []( https://github.com/mapbox/node-sqlite3/wiki/Building-for-Android) wiki but I'm not able to understand how to do it. I'm a very newbie with github and npm so I can't understand some things.
I have a package to clone from github (gekko from askmike);
Before putting into terminal npm install --only=production after the cloning, I have understand that I must:
1.  download the package of sqlite3 (I downloaded the zip file from github: node-sqlite3-master.zip);
2.  then, where should I put the sqlite3 unzipped files? Into what directory of the cloned package (there isn't a node-modules directory after the cloning process... the directory is created after the install process)?
3.  After this, I have to open the file node.gyp but info the cloned directory, there isn't any *.gyp file...
only after the install process a node-pre-gyp file exist and isn't a node-gyp file like the wiki.
So I'm very confused!
